stack/indexer/deb/build_package.sh

```diff
@@ -50,6 +50,15 @@ build_deb() {
             base_cmd+="--reference ${reference}"
         fi
         ../base/generate_base.sh -s ${outdir} -r ${revision} ${base_cmd}
+    else
+        version=$(cat ${current_path}/../../../VERSION)
+        basefile="${outdir}/wazuh-indexer-base-${version}-${revision}-linux-x64.tar.xz"
+        if test -f "${basefile}"; then
+            echo "Building using base file: ${basefile}"
```

Comment:
No need to add an extra log. In fact: only handle if the file does not exist

Reply:
The log is useful to let the user know which file is exactly being used, in case multiple files are located in the folder.